### PR TITLE
Update university domain list

### PR DESCRIPTION
**Pull request by Qt-nhannguyen**
- QTBSI-448: Add new institutions Vol. 25
- QTBSI-463: Add new institutions Vol. 26
- QTBSI-488: Add new institutions Vol. 27

### CODE REVIEW

**Comment by Qt-nhannguyen** [comment]
@ollipuu I have done the website URL, 2-alpha country code, and grammar check for all the new educational institutions. Your double-check would be helpful 😄 .

File: world_universities_and_domains.json [hunk 1]
```diff
@@ -127639,5 +127645,1158 @@
...
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "sgei.org"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "www.andhrauniversity.edu.in"
```
Comment (ollipuu) [inline world_universities_and_domains.json:128456]: Really minor thing, but protocol missing here

Comment (Qt-nhannguyen) [reply world_universities_and_domains.json:128456]: :) that is the missing one. Thank you!

**Review by ollipuu — APPROVED** [review]
Phew, looks good 😮‍💨

**Comment by Qt-nhannguyen** [comment]
> Phew, looks good 😮‍💨

The fact check for these educational institutions is taking much more time than we might expected. At least, from my point of view, making these updates consumes the time of 4 people (Emilia Valkonen-Damjanovic, Petra, you, and me).

I will make a plan to create a tool for Emilia to take care of those.
A rough plan would be:
- Front-end with good input validation for website URL and domain (URL must include HTTP(S) protocol for example)
- Backend with exposed REST API for qt-form service to call.

This would save us a lot of time from many people I believe. Let me raise this topic to our Teams channel

File: world_universities_and_domains.json (hunk 1 above)
Comment (Qt-nhannguyen) [reply world_universities_and_domains.json:128456]: Found out this one is a duplicated university :). Will fix it.

Comment (Qt-nhannguyen) [reply world_universities_and_domains.json:128456]: You can see the fix in this commit 50e2f3527258f8be170102599a4746c31af0b6a6

**Pull request merged**